ozaria/engine/cinematic/systems/autoplay.js

```diff
@@ -0,0 +1,17 @@
+import { getWaitUserInput } from '../../../../app/schemas/models/selectors/cinematic'
+import { SyncFunction } from '../commands/commands'
+
+export default class Autoplay {
+  constructor () {
+    this.autoplay = false
+  }
+
+  parseDialogNode (dialogNode) {
```

Comment:
Where do we use this method?

Reply:
Great question! This is implicit behavior of a registered Cinematic system.

`parseDialogNode` is one of the interfaces of a Cinematic System.

I think it might be good to create an empty System class that these systems extend to make this more explicit. This empty System class would have the interface that a Cinematic system should have.

A cinematic system automatically works if it has this method and is registered to the `cinematicController`. This is due to the cinematic parser method iterating over all `systems` and calling this method with the right arguments if it is implemented.